src/wp-includes/option.php

```diff
@@ -247,6 +247,95 @@ function get_option( $option, $default_value = false ) {
 	return apply_filters( "option_{$option}", maybe_unserialize( $value ), $option );
 }
 
+/**
+ *
+ * @since 6.6.0
+ *
+ * @global wpdb $wpdb WordPress database abstraction object.
+ *
+ * @param string[] $options An array of option names to be loaded.
+ */
+function wp_prime_network_option_caches( $network_id = null, array $options = array() ) {
```

Comment:
Do you think it's worth having `wp_prime_network_option_caches( array $options = [] )` as a wrapper for consistency with the other option functions?

Reply:
The documentation is missing for new param.